.pre-commit-config.yaml

```
@@ -81,3 +81,12 @@ repos:
         language: system
         stages: [commit]
         require_serial: true
+
+      - id: mlver
+        name: check
```

```suggestion
        name: ml-package-version-consistency
```
Just to make the intention of the hook more visible, assuming there will be no additional message is shown about why the file is updated.

---

Thanks for the catch, sounds good!